### PR TITLE
Added detailed information about order and menu in personal account (mobile)

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: frontend/src/components/account/Orders/components/OrderItemMobile/OrderItemMobile.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    const fetchData = async () => {
+      try {
+        const response = await getOrderById(orderDetails);
+        setOrder(response);
+        setQuantity(response.orderItems.length);
+      } catch (error) {
+        console.error(
+          'Помилка при отриманні замовлень коричтувача: ',
+          error.message,
+        )();
```
Comment (halynarydasova) [inline frontend/src/components/account/Orders/components/OrderItemMobile/OrderItemMobile.jsx:33]: same here, delete extra ()

File: frontend/src/components/account/Orders/components/OrdersMobile/OrdersMobile.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ICONS } from '../../../../ui-kit/icons';
+import styles from './OrdersMobile.module.scss';
+import { useNavigate } from 'react-router-dom';
+
+const OrdersMobile = () => {
+  const navigate = useNavigate();
```
Comment (halynarydasova) [inline frontend/src/components/account/Orders/components/OrdersMobile/OrdersMobile.jsx:7]: Delete this component. You have this info in TitleMobile

File: frontend/src/components/account/Orders/components/OrderItemMobile/OrderItemMobile.jsx (hunk 1 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/components/OrderItemMobile/OrderItemMobile.jsx:33]: Done

File: frontend/src/components/account/Orders/OrdersList.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  const fetchData = async () => {
+    try {
+      const response = await getUsersOrders();
+      setData(response);
+    } catch (error) {
+      console.error(
+        'Помилка при отриманні замовлень коричтувача: ',
+        error.message,
+      )();
```
[the remark this answers is not in the document]
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/OrdersList.jsx:18]: done

File: frontend/src/components/account/Orders/components/OrdersMobile/OrdersMobile.jsx (hunk 2 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/components/OrdersMobile/OrdersMobile.jsx:7]: Done

File: frontend/src/components/account/AccountMobileMenu/AccountMobileMenu.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import SideMenu from '../SideMenu';
+import React from 'react';
+
+const AccountMobileMenu = ({ onCloseMobileMenu }) => {
```
[the remark this answers is not in the document]
Comment (Svitlana-G) [reply frontend/src/components/account/AccountMobileMenu/AccountMobileMenu.jsx:4]: Not anymore. Removed.

**Pull request merged**